src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -56,6 +56,7 @@ interface CourseRunEnrollmentProps {
     id: number;
     resource_link: string;
     priority: number;
+    starts_in_message: string;
```

Comment:
Since this property can be null if the course started, the correct type should be:
```suggestion
    starts_in_message: Nullable<string>;
```